src/main/java/practicalities/items/ItemMagnet.java

```diff
@@ -64,6 +67,8 @@ public void onUpdate(ItemStack item, World world, Entity entity, int i, boolean
 				this.distanceFromPlayer).iterator();
 		while (iterator.hasNext()) {
 			EntityItem itemToGet = (EntityItem) iterator.next();
+			if( PolarityNegatorManager.isEntityCloseToNegator(itemToGet) )
```

Comment:
What do you think of hooking into the itempickup event for this? That way it would work for other magnets